lib/dal/src/workspace_snapshot/migrator.rs

```diff
@@ -108,16 +120,20 @@ impl SnapshotGraphMigrator {
                 }
 
                 let snapshot_address = change_set.workspace_snapshot_address;
-                info!(
+                trace!(
                     "Migrating snapshot {} for change set {} with base change set of {:?}",
-                    snapshot_address, change_set_id, change_set.base_change_set_id,
+                    snapshot_address,
+                    change_set_id,
+                    change_set.base_change_set_id,
                 );
 
                 let new_snapshot = match self.migrate_snapshot(ctx, snapshot_address).await {
                     Ok(new_snapshot) => new_snapshot,
                     Err(err) => {
                         let err_string = err.to_string();
-                        if err_string.contains("missing from store for node") {
+                        if err_string.contains("missing from store for node")
+                            || err_string.contains("workspace snapshot graph missing at address")
```

Comment:
This is the current "fix" for the error that was blocking migrations: [trace](https://ui.honeycomb.io/system-initiative/environments/tools-production/datasets/sdf/result/D93GkGdxAyY/trace/xqS2GbSVofM?fields[]=s_name&fields[]=s_serviceName&span=a25281cc6e2b076c)

Delete changesets with unreachable workspace snapshots